integration_tests/__tests__/moduleNameMapper-test.js

```diff
@@ -0,0 +1,18 @@
+const runJest = require('../runJest');
```

Comment:
Please add copyright headers to every JS file. I mean just like this:

```
/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 */
```
Without `'use strict';` (rebase to master to see why) without `@flow` (we're currently not type-checking our tests) and without `@emails...`